.github/workflows/validate-pr-actions.yaml

```diff
@@ -0,0 +1,108 @@
+---
+name: Validate Github Actions
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        type: string
+        required: false
+  pull_request:
+    paths:
+      - actions/**
+      - tests/test_resources/mpyl_config.yml
+      - tests/test_resources/repository/changed_files.json
+      - tests/projects/job/**
+
+# If multiple commits are pushed simultaneously, only build the last one.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+
+  health-check:
+    name: ▶️ Run actions/health-check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.2.1
+
+      - name: Point action.yaml to a new version
+        if: inputs.image != ''
+        env:
+          IMAGE: docker://${{ inputs.image }}
+        run: yq -i '.runs.image = strenv(IMAGE)' actions/health-check/action.yaml
+
+      - uses: ./actions/health-check
+        with:
+          config-path: tests/test_resources/mpyl_config.yml
```

Comment:
Actually, for these kind of actions or complete test runs we always use the config files in the root of this repo. The ones you are pointing at now are used specifically for the unit tests (and are usually more hacky/incomplete for that reason)

Reply:
I'm not sure I understand, currently there is no `mpyl_config.yml` in the root of this repo.

Reply:
I meant the example file. Extra benefit is that it forces us to keep it (the example) up to date

Reply:
Check, done: https://github.com/Vandebron/gh-mpyl/pull/26/commits/d7e1cf35c216af959e5e495776874880e5ea44dc